app/controllers/profiles_controller.rb

```diff
@@ -38,7 +38,7 @@ def create
       Agreement.create!(profile_id: @profile.id, form_item_id: 8, value: 1) if agreement_params['redhat_require_email_cndt2022']
       Agreement.create!(profile_id: @profile.id, form_item_id: 9, value: 1) if agreement_params['redhat_require_tel_cndt2022']
 
-      ProfileMailer.registered(@profile, @conference).deliver_later
+      #TODO: ProfileMailer.registered(@profile, @conference).deliver_later
```

Comment:
何のTODOか分からないけど開発時にコメントアウトして戻し忘れた？

```suggestion
      ProfileMailer.registered(@profile, @conference).deliver_later
```

Reply:
あーしもた